flake/flake-modules/auto.nix

```diff
@@ -0,0 +1,72 @@
+{ lib, config, ... }:
+let
+  cfg = config.nixvim;
+in
+{
+  options.nixvim = {
+    packages = {
+      enable = lib.mkOption {
+        type = lib.types.bool;
+        description = "Whether to automatically install packages for each nixvimConfiguration.";
+        default = false;
```

Comment:
Are we sure to default false for it?

Reply:
I wouldn't say I'm sure...

My rationale was;  adding `packages` _by default_ may be more controversial than adding `checks` _by default_

Reply:
I can't manage to convince myself one way or the other, so I'll defer to anyone with an opinion

Reply:
> Why wouldn't someone define a nixvim config in his flake without wanting to have the `packages` output ?

Maybe they have several nixvim configurations and only want package outputs for some of them.

Maybe they want to use the nixvim configuration packages _only_ within other configurations (NixOS, home-manager).

Maybe they want bespoke/different names for the package attrs, and `nameFunction` is impractical for what they want to achieve.

I'm not _convinced_ that we should be messing with users flake outputs without them being aware, i.e. having opted in.

`checks` feel less special than `packages`, but maybe both should be default-false for sanity & consistency?